app-modules/panel-app/src/Livewire/MyProfile/CandidatePreferences.php

```
@@ -49,10 +50,30 @@ public function form(Schema $schema): Schema
+                        'x-mask:dynamic' => "
+                                \$input.includes('.')
+                                    ? \$money(\$input, '.', ',', 2)
+                                    : \$money(\$input, '.', ',', 0)
+                            ",
+                    ])
+                    ->dehydrateStateUsing(fn ($state) => $state ? (float) str_replace(',', '', $state) : null
+                    )
+                    ->formatStateUsing(fn ($state) => $state ? number_format((float) $state, 0, '.', ',') : null
+                    )
+                    ->rules([
+                        fn () => function (string $attribute, $value, Closure $fail): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)